iphone/Classes/TiUIScrollableViewProxy.m

```diff
@@ -71,7 +71,7 @@ -(NSUInteger)viewCount
 
 -(void)setViews:(id)args
 {
-#ifndef TI_USE_AUTOLAYOUT
+#ifdef TI_USE_AUTOLAYOUT
```

Comment:
Hey @nuno, sounds critical. Can you describe some more details on the related ticket?